Project.toml

```
@@ -16,16 +17,18 @@ URIs = "5c2747f8-b7ea-4ff2-ba2e-563bfd36b1d4"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
-Base64 = "1"
-Dates = "1"
+Base64 = "<0.0.1, 1"
+Dates = "<0.0.1, 1"
```

I think this is not needed - at least it was not needed for 2.2.0 a few days ago:
```suggestion
Base64 = "1"
Dates = "1"
```

---

So, if there is a package PkgA that depends on FHIRClient, and if PkgA performs any Pkg operations in its test suite, then PkgA's test suite will fail unless we include the `<0.0.1`.

Since we don't know about any private packages out there that depend on FHIRClient, to be safe, we need to include the `<0.0.1`.

---

I just noticed that there was another update of https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958 that mentions this Pkg problem. My comment was based on the fact that we only support Julia >= 1.4 in FHIRClient, and hence based on the previous announcements and the successful registration of release 2.2.0 I assumed that we do not have to add `<0.0.1` in FHIRClient. It seems quite unfortunate and undesirable that apparently it is possible to accidentally break other packages even if FHIRClient itself is completely fine and it's (currently) not possible to notice these problems. I think that most packages that were updated based on the previous announcements are not aware of this fact (most commonly, I've seen people just adding `... = "1"` entries).

---

Yeah, my thinking now is that we just need to tell everyone to just use `"<0.0.1, 1"`.

---

We might need to modify AutoMerge so that it fails if someone doesn't have the `<0.0.1` for a stdlib.